app/models/shipit/deploy_spec/lerna_discovery.rb

```diff
@@ -41,7 +41,7 @@ def lerna_json
         file('lerna.json')
       end
 
-      def lerna_version
+      def repo_version
```

Comment:
I'd rather stick with `lerna_version`.  All of these methods [get munged into a single Ruby module](https://github.com/Shopify/shipit-engine/blob/f0dcc4aa4eac152bf437b562fc4857752ed6d4c8/app/models/shipit/deploy_spec/file_system.rb#L3), so debugging is easier when generic names aren't used.

This change will make sense once `Shipit::DeploySpec::FileSystem` is refactored.